[12.0][ADD] rma: new module (#153)
OCA/rma: rma/README.rst

===

> RMA

Reviewer: Return Merchandise Authorization Management

Author: To do that I have to change the name in __manifest__.py and that causes the full name of the app not to show in the kanban view. Do i do it anyway?

Author: ![Screenshot 2020-06-10 11:17:43](https://user-images.githubusercontent.com/38267832/84286004-1ea04a80-ab0c-11ea-9c0e-9ad15d364505.png)

Reviewer: Yes, who look at that view? :stuck_out_tongue_closed_eyes:

    :target: https://odoo-community.org/page/development-status
    :alt: Beta
.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
    :alt: License: AGPL-3
.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Frma-lightgray.png?logo=github
    :target: https://github.com/OCA/rma/tree/12.0/rma
    :alt: OCA/rma
.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
    :target: https://translation.odoo-community.org/projects/rma-12-0/rma-12-0-rma
    :alt: Translate me on Weblate
.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
    :target: https://runbot.odoo-community.org/runbot/145/12.0
    :alt: Try me on Runbot

|badge1| |badge2| |badge3| |badge4| |badge5| 

This module allows you to manage `Return Merchandise Authorization (RMA)
<https://en.wikipedia.org/wiki/Return_merchandise_authorization>`_.
RMA documents can be created from scratch, from a delivery order or from
an incoming email. Product receptions and returning delivery operations
of the RMA module are fully integrated with the Receipts and Deliveries

> Operations of Odoo inventory core module. It also allows you to generates

Reviewer: ```suggestion
Operations of Odoo inventory core module. It also allows you to generate
```

refunds in the same way as Odoo generate it. Moreover, you can access to all

> RMA web pages form the portal.

Reviewer: ```suggestion
refunds in the same way as Odoo generates it.

Besides, you have full integration of the RMA documents in the customer portal.
```

**Table of contents**

.. contents::
   :local:

Configuration
=============

If you want RMAs to be created from incoming emails, you need to:

#. Go to *Settings > General Settings*.
#. Check 'External Email Servers' checkbox under *Discuss* section.
#. Set an 'alias domain' and an incoming server.

> #. Got to *RMA > Configuration > RMA Team* and select a team or create a new

Reviewer: ```suggestion
#. Go to *RMA > Configuration > RMA Team* and select a team or create a new
```

Usage
=====

To use this module, you need to:

#. Go to *RMA > Orders* and create a new RMA.

   .. image:: https://raw.githubusercontent.com/OCA/rma/12.0/rma/static/images/menu.png
      :alt: Menu

#. Select a partner, an invoice address, select a product
   (or select a picking and a move instead), write a quantity, fill the rest
   of the form and click on 'confirm' button in the status bar.

   .. image:: https://raw.githubusercontent.com/OCA/rma/12.0/rma/static/images/create_confirm.png
      :alt: Create confirm

#. You will see an smart button labeled 'Receipt'. Click on that button to see
   the reception operation form.

   .. image:: https://raw.githubusercontent.com/OCA/rma/12.0/rma/static/images/access_to_receipt.png
      :alt: Access to receipt

#. If everything is right, validate the operation and go back to the RMA to
   see it in a 'received' state.

   .. image:: https://raw.githubusercontent.com/OCA/rma/12.0/rma/static/images/validate_receipt.png
      :alt: Validate receipt

#. Now you are able to generate a refund, generate a delivery order to return
   to the customer the same product or another product as a replacement, split

>    the RMA by extracting a part of a remaining quantity to another RMA, preview

Reviewer: ```suggestion
   the RMA by extracting a part of the remaining quantity to another RMA, preview
```
But put this as subpoints instead everything together in the same main point

>    the RMA in the website. All of this operations can be done by clicking on

Reviewer: ```suggestion
   the RMA in the website. All of these operations can be done by clicking on
```

   the buttons in the status bar.

   .. image:: https://raw.githubusercontent.com/OCA/rma/12.0/rma/static/images/operation_on_received.png
      :alt: Operation on received

>    * If you click on 'Refund' button a refund will be create and it will be

Reviewer: ```suggestion
   * If you click on 'Refund' button, a refund will be created, and it will be
```

     accessible via the smart button labeled Refund. The RMA will be set

>      automatically to 'refunded' state when the refund is validated.

Reviewer: ```suggestion
     automatically to 'Refunded' state when the refund is validated.
```

     .. image:: https://raw.githubusercontent.com/OCA/rma/12.0/rma/static/images/refunded_rma.png
      :alt: Refunded RMA

>    * But, if you click on 'Replace' or 'Return to customer' button instead,

Reviewer: ```suggestion
   * If you click on 'Replace' or 'Return to customer' button instead,
```

     a popup wizard will guide you to create a Delivery order to the client
     and this order will be accessible via the smart button labeled Delivery.
     The RMA will be set automatically to 'delivered' state when the RMA

>      quantity is delivery in one or more 'done' delivery orders.

Reviewer: ```suggestion
     The RMA will be set automatically to 'Delivered' state when the RMA
     quantity is equal or lower than the quantity in done delivery orders linked to it.
```
